scripts/logbook.py

```diff
@@ -48,31 +49,32 @@ def categorization(process_name):
         #"Cluster Generation (Illumina SBS) 4.0" : {"dest_file" : ["cBot"], "instrument" : ["lims_instrument"], "details" : [""]},
+        "Denature, Dilute and Load Sample (MiSeq) 4.0" : {"dest_file" : ["MiSeq"], "instrument" : ["udf_Instrument Used"], "details": [{"udf_Flowcell ID" : "", "udf_RGT#s" : ""}]},
         "End Repair, A-Tailing and Adapter Ligation (SS XT) 4.0" : {"dest_file" : ["Bravo", "PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]},
         "End repair, size selection, A-tailing and adapter ligation (Lucigen NxSeq DNA) 4.0" : {"dest_file" : ["Bravo"], "instrument" : ["lims_instrument"], "details" : [""]},
         "End repair, size selection, A-tailing and adapter ligation (TruSeq DNA Nano) 4.0" : {"dest_file" : ["Bravo"], "instrument" : ["lims_instrument"], "details" : [""]},
-        "End repair, size selection, A-tailing and adapter ligation (TruSeq PCR-free DNA) 4.0" : {"dest_file" : ["Bravo", "PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]},
+        "End repair, size selection, A-tailing and adapter ligation (TruSeq PCR-free DNA) 4.0" : {"dest_file" : ["Bravo", "PCR"], "instrument" : ["lims_instrument", "udf_PCR machine"], "details" : ["", ""]},
         "End repair, A-tailing and adapter ligation (TruSeq RNA) 4.0" : {"dest_file" : ["Bravo", "PCR"], "instrument" : ["lims_instrument", "udf_PCR Machine"], "details" : ["", ""]},
         "Enrich DNA fragments (Nextera) 4.0" : {"dest_file" : ["PCR"], "instrument" : ["lims_instrument"], "details" : [""]},
         "Enrich DNA fragments (TruSeq DNA) 4.0" : {"dest_file" : ["PCR", "Bravo"], "instrument" : ["lims_instrument", "udf_Bravo"], "details" : ["", ""]},
         "Enrich DNA fragments (TruSeq RNA) 4.0" : {"dest_file" : ["PCR"], "instrument" : ["lims_instrument"], "details" : [""]},
         "Fragment Analyzer QC (DNA) 4.0" : {"dest_file" : ["FragmentAnalyzer"], "instrument" : ["default"], "details": [{"udf_Lot no: Fragment Analyzer Reagents" : ""}]},
         "Fragment Analyzer QC (Library Validation) 4.0" : {"dest_file" : ["FragmentAnalyzer"], "instrument" : ["default"], "details": [{"udf_Lot no: Fragment Analyzer Reagents" : ""}]},
         "Fragment Analyzer QC (RNA) 4.0" : {"dest_file" : ["FragmentAnalyzer"], "instrument" : ["default"], "details": [{"udf_Lot no: Fragment Analyzer Reagents" : ""}]},
-        "Fragment DNA (ThruPlex)" : {"dest_file" : ["Covaris"], "instrument" : ["lims_instrument"], "details" : ["udf_Lot no: Covaris tube"]},
-        "Fragment DNA (TruSeq DNA) 4.0" : {"dest_file" : ["Covaris"], "instrument" : ["lims_instrument"], "details" : ["udf_Lot no: Covaris tube"]},
+        "Fragment DNA (ThruPlex)" : {"dest_file" : ["Covaris"], "instrument" : ["lims_instrument"], "details" : [{"udf_Lot no: Covaris tube" : ""}]},
+        "Fragment DNA (TruSeq DNA) 4.0" : {"dest_file" : ["Covaris"], "instrument" : ["lims_instrument"], "details" : [{"udf_Lot no: Covaris tube" : ""}]},
```

Comment:
There is no purpose of having a `dict` if the `key` has empty `value`. I would do something like this

```
"Fragment DNA (TruSeq DNA) 4.0" : {"dest_file" : ["Covaris"], "instrument" : {"lims_instrument": ["udf_Lot no: Covaris tube"]}
```

if you think sometimes the instruments can have no details than for those cases you can do something like

```
"End repair, size selection, A-tailing and adapter ligation (TruSeq PCR-free DNA) 4.0" : {"dest_file" : ["Bravo", "PCR"], "instrument" : {"lims_instrument" : [], "udf_PCR machine": []}}
```

In this way you can skip the check `if record["details"][instrument_number] is not '':` downstream and just directly iterate like `for item in record["instrument"][instrument_name]:`